components/dataViews/MultisigMembers.js

```diff
@@ -1,70 +1,54 @@
 import React from "react";
 
+import HashView from "./HashView";
 import StackableContainer from "../layout/StackableContainer";
-import { abbreviateLongString } from "../../lib/displayHelpers";
 
-const dummyMembers = [
-  {
-    nickname: "Tolla",
-    pubkey: "AqsujWsohxvLS8B6ANf54D9qtIhAtQ2ISptBmXGUZVIN",
-    address: "cosmos1j8z4cfgpza4qa33pl02y84n0mdm8n7xzqdwlse",
-  },
-  {
-    nickname: "Yamman",
-    pubkey: "AqsujWsohxvLS8B6ANf54D9qtIhAtQ2ISptBmXGUZVIN",
-    address: "cosmos145mr4l98w2x96utkcayvtqaag79u7mpyane7q7",
-  },
-  {
-    nickname: "Wallace",
-    pubkey: "AqsujWsohxvLS8B6ANf54D9qtIhAtQ2ISptBmXGUZVIN",
-    address: "cosmos1t5u0jfg3ljsjrh2m9e47d4ny2hea7eehxrzdgd",
-  },
-];
 const MultisigMembers = (_props) => (
   <StackableContainer lessPadding>
-    <h2>Members</h2>
-    <ul className="meta-data">
-      {dummyMembers.map((member) => (
-        <li key={member.address}>
-          <div className="nickname">
-            <label>Nickname:</label>
-            <div className="info">{member.nickname}</div>
-          </div>
-          <div className="address">
-            <label>Address:</label>
-            <div className="info">{abbreviateLongString(member.address)}</div>
-          </div>
-        </li>
-      ))}
-    </ul>
+    <div className="meta-data">
+      <div>
+        <h2>Threshold</h2>
+        <div className="info threshold">{_props.threshold}</div>
```

Comment:
Could you rename `_props` to `props`? The leading underscore is for silencing the linter in case of unused variables.

Reply:
ok just pushed this change